bundle/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>com.adobe.acs</groupId>
         <artifactId>acs-aem-commons</artifactId>
-        <version>4.2.1-SNAPSHOT</version>
```

Comment:
Individual PRs should not modify the version in the pom files.  The version is already SNAPSHOT so there is no need to up it for testing.

Reply:
Restored version